tests/indico_wrapper/test_teach.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+import os
+
+from indico_toolkit.indico_wrapper import Teach
+from tests.conftest import FILE_PATH
+
+
+@pytest.fixture(scope="module")
```

Comment:
looks like a pretty complete fixture of integration tests.  would it be worth investing in a mocked indico client so we can do more unit level testing for modules like this?